tests/integration/mwss/test_mwss_submission_data.py

```diff
@@ -0,0 +1,326 @@
+            'four-weekly-pay-significant-changes-pay-rates-increase-percent-employees-answer': '200%'
+        }
+
+        self.post(form_data)
+        self.post(post_data={'four-weekly-pay-significant-changes-industrial-action-answer': ['Yes']})
+        self.post(post_data={'four-weekly-pay-significant-changes-other-answer': ['Yes']})
+        self.post(post_data={'four-weekly-pay-significant-changes-other-specify-answer': 'four Weekly Pay'})
+
+        # We are in Five Weekly Pay section
+        self.assertInPage('>Five Weekly Pay</')
+        self.assertInPage('The next section covers Five Weekly Pay.')
+        self.post(action='save_continue')
+
+        self.post(post_data={'five-weekly-pay-paid-employees-answer': '10'})
+        self.post(post_data={'five-weekly-pay-gross-pay-answer': '10'})
+
+        self.post(post_data={'five-weekly-pay-breakdown-arrears-answer': '2',
+                             'five-weekly-pay-breakdown-prp-answer': '3'})
+
+        self.post(post_data={'five-weekly-pay-significant-changes-paid-employees-answer': ['Yes']})
+        self.post(post_data={'five-weekly-pay-significant-changes-redundancies-answer': ['Yes']})
+        self.post(post_data={'five-weekly-pay-significant-changes-temp-employees-answer': ['Fewer temporary staff']})
+        self.post(post_data={'five-weekly-pay-significant-changes-gross-pay-answer': ['Yes']})
+        self.post(post_data={'five-weekly-pay-significant-changes-overtime-answer': ['More overtime']})
+        self.post(post_data={'five-weekly-pay-significant-changes-pay-rates-answer': ['Yes']})
+
+        form_data = {
+            # Percentage increase of pay rate
+            'five-weekly-pay-significant-changes-pay-rates-increase-percent-increase-answer': '10',
+            # figures are back dated, from when
+            'five-weekly-pay-significant-changes-pay-rates-increase-date-from-answer-day': '01',
+            'five-weekly-pay-significant-changes-pay-rates-increase-date-from-answer-month': '8',
+            'five-weekly-pay-significant-changes-pay-rates-increase-date-from-answer-year': '2016',
+            # percentage or number of five-weekly paid employees who received this new pay rate?
+            'five-weekly-pay-significant-changes-pay-rates-increase-percent-employees-answer': '100'
+        }
+
+        self.post(form_data)
+        self.post(post_data={'five-weekly-pay-significant-changes-industrial-action-answer': ['Yes']})
+        self.post(post_data={'five-weekly-pay-significant-changes-other-answer': ['Yes']})
+        self.post(post_data={'five-weekly-pay-significant-changes-other-specify-answer': 'five Weekly Pay'})
+
+        # We are in Five Weekly Pay section
+        self.assertInPage('>General Comments</')
+        self.assertInPage('The next section covers general comments about this survey.')
+        self.post(action='save_continue')
+
+        self.post(post_data={'general-comments-answer': 'General Comments'})
```

Comment:
Maybe one assert to ensure you're on the summary/confirmation page after this post ?